validators/overlays/all/classes/_rules/check_mdattr.xsl

```diff
@@ -39,7 +39,7 @@
         </xsl:call-template>
     </xsl:template>
     <xsl:template match="md:Extensions[mdattr:EntityAttributes]
-        [not(parent::md:EntityDescriptor)][not(parent::md:EntitiesDescriptor)]">
+        [not((parent::md:EntityDescriptor) or (parent::md:EntitiesDescriptor))]">
```

Comment:
Are those inner parentheses `(parent::x)` actually required, or could they be omitted?

Reply:
> I do tend to use too many parentheses. I will take a look.

Reply:
I'm normally in favour of using more rather than less, for clarity (and when I can't remember operator precedence). In this case I think it actually makes things less clear, as the inner parentheses are around single terms. It's like `foo((a) or (b))` and runs the risk of visually attaching the first term to the function.

Reply:
Yes, this works fine without the extra parentheses. And, it does indeed look clearer in this case. So I will commit this.